core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -209,6 +209,11 @@ abstract class RDD[T: ClassTag](
    */
   def cache(): this.type = persist()
 
+  def materialize(): RDD[T] = withScope{
```

Comment:
Please fix the PR title. This is RDD API, not a `Dataframe` API.

Reply:
This PR also includes `Dataset.materialize`, which calls this RDD version.